src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Projects/NuGetProjectFactory.cs

```diff
@@ -23,6 +24,11 @@ public sealed class NuGetProjectFactory
         private readonly IVsProjectThreadingService _threadingService;
         private readonly Common.ILogger _logger;
 
+        // Reason it's lazy<object> is because we don't want to load any CPS assemblies untill
+        // we're really going to use any of CPS api. Which is why we also don't use nameof or typeof apis.
```

Comment:
typo: untill

could be made more clear also

Reply:
It was the same comment before and I don't know if I can make it more clear.